client/packages/requisitions/src/ResponseRequisition/DetailView/ResponseLineEdit/ResponseLineEdit.tsx

```diff
@@ -93,6 +101,12 @@ export const ResponseLineEdit = ({
             value={String(currentItem?.defaultPackSize)}
           />
         )}
+        {displayVaccinesInDoses && currentItem?.doses ? (
+          <InfoRow
+            label={t('label.doses-per-unit')}
+            value={String(currentItem?.doses)}
+          />
+        ) : null}
```

Comment:
FYI, in both modals - days out of stock is showing in vials 👀 

![Screenshot 2025-06-05 at 10 37 31 AM](https://github.com/user-attachments/assets/abcba564-8375-4420-b723-32087ca0c1c7)

Reply:
Mine is not...?

![Screenshot 2025-06-05 at 11 57 06 AM](https://github.com/user-attachments/assets/a9e44d22-0524-418f-b719-8ad507f055ea)
![Screenshot 2025-06-05 at 11 57 23 AM](https://github.com/user-attachments/assets/ee489790-d15e-459f-a254-9cc58f7bd60d)

Reply:
Oh i mean, its `days`, shouldn't be measured in packs/units 😅